docs/src/reference/compilation.rst

```diff
@@ -490,7 +490,7 @@ will fall back to loading the source modules instead.
 Note that it is not recommended to let :mod:`pyximport` build code
 on end user side as it hooks into their import system.  The best way
 to cater for end users is to provide pre-built binary packages in the
-`wheel <https://wheel.readthedocs.io/>`_ packaging format.
+`wheel <https://wheel.readthedocs.io/en/stable/>`_ packaging format.
```

Comment:
AFAIK, this kind of redirect depends on the user's browser and its language settings. While there isn't anything but an English documentation _right now_, I wouldn't want to rule out there ever being a translation, so this should be kept as it is.